modules/Hoot/managers/api.js

```diff
@@ -564,7 +564,9 @@ export default class API {
             } )
             .catch( err => {
                 return Promise.reject( {
-                    data: err.data,
+                    data: {
+                        details: err.data.commandDetail[0].stderr
```

Comment:
Should we do the right thing and do a `join(';')` or something instead of `[0]`?